Two parts of the kernel want the headers of the Active Template Library,
which are a part of Visual Studio that this machine does not have and that cannot
be written to.  The parts are the embedding of objects and the accessibility of
Windows; the kernel of the documents needs neither.

diff --git a/embedserv/Module_embedserv.mk b/embedserv/Module_embedserv.mk
index 505efa5c4..be8ffa2c4 100644
--- a/embedserv/Module_embedserv.mk
+++ b/embedserv/Module_embedserv.mk
@@ -13,7 +13,6 @@ $(eval $(call gb_Module_Module,embedserv))
 ifeq ($(OS),WNT)
 
 $(eval $(call gb_Module_add_targets,embedserv,\
-	Library_emser \
 	Library_inprocserv \
 ))
 
diff --git a/winaccessibility/Module_winaccessibility.mk b/winaccessibility/Module_winaccessibility.mk
index 2421164b3..8846ffb83 100644
--- a/winaccessibility/Module_winaccessibility.mk
+++ b/winaccessibility/Module_winaccessibility.mk
@@ -12,10 +12,6 @@ $(eval $(call gb_Module_Module,winaccessibility))
 ifeq ($(OS),WNT)
 ifeq ($(COM),MSC)
 $(eval $(call gb_Module_add_targets,winaccessibility,\
-	WinResTarget_uacccom \
-	CustomTarget_ia2_idl \
-	Library_uacccom \
-	Library_winaccessibility \
 ))
 endif # COM=MSC
 endif # WNT
